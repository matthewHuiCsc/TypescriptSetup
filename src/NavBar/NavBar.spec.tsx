declare var jest, describe, it, expect, test;

/* T_T why? */
import * as React from 'react';
/* Why can't I do `export default NavBar` instead? */
import { NavBar } from './index';

/* Wha? Why do I need to use a require in this case instead? */
let renderer = require('react-test-renderer');

test('NavBar renders correctly', () => {
  const tree = renderer.create(
    <NavBar></NavBar>
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
