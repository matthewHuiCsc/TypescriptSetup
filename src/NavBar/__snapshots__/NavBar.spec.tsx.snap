exports[`test NavBar renders correctly 1`] = `
<div>
   I am a nav bar! 
</div>
`;
